plugins/advisors/black-duck/src/main/kotlin/BlackDuck.kt

```diff
@@ -158,10 +158,10 @@ class BlackDuck(
         }
 
         if (origins.isEmpty()) {
-            logger.info { "No origin found for package '${pkg.id.toCoordinates()}'." }
+            logger.info { "No origin found for package '${pkg.id.toCoordinates()}', ${pkg.requestParam}." }
```

Comment:
I find it a bit unusual to use a comma here, as it's not an enumeration, but an alternative way to refer to the same package. So maybe use parentheses as in `(${pkg.requestParam})` instead?

Reply:
Did you forget to push this change?